### PR TITLE
New option 'only detected rules'

### PR DESCRIPTION
**Pull request by salvorusso**
The main change is to allow users to include only the detected rules in the report, making the Known Security Rules section more compact and concise.

### Key Changes:

New Option: We have introduced a new command-line option, `--only-detected-rules`, which, when used during report generation, will include only the rules that have been detected.

### Benefits:

**Enhanced Clarity**: By including only the detected rules in the report, the Known Security Rules section will become significantly more concise. This simplifies the report and ensures that users can focus on the most critical information without unnecessary clutter.

**Improved Readability**: Users will appreciate the improved readability of the report, which will make it easier to identify and prioritize security issues.

How to Use:

To utilize this new feature, simply include the `--only-detected-rules` option when generating the report. For example:
```bash
# Generate report example
sonar-report \
  --sonarurl="https://sonarcloud.io" \
  --sonarcomponent="soprasteria_sonar-report" \
  --sonarorganization="sopra-steria" \
  --project="Sonar Report" \
  --application="sonar-report" \
  --release="1.0.0" \
  --branch="master" \
  --only-detected-rules \
  --output="samples/sonar-report_sonar-report.html"
```

### CODE REVIEW

**Comment by thecampagnards** [comment]
Hello, I can't merge this pr since you rename the project, but I can merge it if you update this pr to have only your feature, if not I will create an issue to implement it later
BR

File: index.ejs [hunk 1]
```diff
@@ -16,69 +16,76 @@
 <body>
```
Comment (thecampagnards) [inline index.ejs:16]: for this you can use `--ejs-file` to use a custom template file

**Review by thecampagnards — CHANGES_REQUESTED** [review]
I finished the review it's good, just revert sonarqube to sonar to merge it
BR

**Comment by salvorusso** [comment]
I have opened a new PR with the requested feature, thanks

**Pull request closed without merging**